rust/parquet/README.md

```diff
@@ -46,6 +46,15 @@ while let Some(record) = iter.next() {
 ```
 See [crate documentation](https://docs.rs/crate/parquet/4.0.0-SNAPSHOT) on available API.
 
+## Upgrading from versions prior to 4.0
```

Comment:
It's fine to add this temporarily in README.md but I think we should put them this alongside other breaking changes from 4.0 (seems there're quite a few of them) in a separate doc eventually. [Something similar](https://github.com/apache/spark/blob/master/docs/sql-migration-guide.md) from Apache Spark.

Reply:
Summarizing breaking changes would definitely be valuable. I don't think I have the time to do so at this moment, but I am very supportive of the idea

Reply:
I'll add a guide as part of https://issues.apache.org/jira/browse/ARROW-12019, to provide examples of how one can start using `LogicalType`.